amp/WEB-INF/src/org/digijava/kernel/ampapi/endpoints/activity/ObjectImporter.java

```diff
@@ -566,6 +569,10 @@ public PossibleValuesCache getPossibleValuesCache() {
         return this.possibleValuesCached;
     }
 
+    public ResourceService getResourceService() {
+        return this.resourceService;
+    }
+
```

Comment:
Please move it to `ActivityImporter` and cast `importer` to `ActivityImporter` in `UUIDValidator`